### PR TITLE
Update async clipboard api

### PR DESCRIPTION
**Pull request by developit**
What's changed, or what was fixed?
- Added `/web` introducing the Asynchronous Clipboard API

**Depends On:** GoogleChrome/samples#571
**Target Live Date:** 2018-03-15

- [x] This has been reviewed and approved by (@ebidel)
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele @ebidel @garykac

### CODE REVIEW

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+{# wf_blink_components: Blink>Input #}
+
+
+# Unblocking Clipboard Access {: .page-title }
+
+{% include "web/_shared/contributors/developit.html" %}
+
+
+Over the past few years, browsers have converged on using
+[document.execCommand] for clipboard interactions. It's great to have a single
```
Comment (petele) [inline src/content/en/updates/2018/03/clipboardapi.md]: wrap document.execCommand in backticks ie: 
```
[`document.execCommand`]
```

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+might a dedicated clipboard API look like if we wanted to address blocking and
+permissions problems?
+
+That's the new [Async Clipboard API], the text-focused portion of which we're
+shipping in Chrome 66.  It's a replacement for execCommand-based copy & paste
+that has a well-defined permissions model and doesn't block the page.  This new
+API also Promises (see what I did there?) to simplify clipboard events and align
+them with the [Drag & Drop API].
+
+<div style="display: flex; justify-content: center">
```
Comment (petele) [inline src/content/en/updates/2018/03/clipboardapi.md]: why wrap this in flex?

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 3]
```diff
@@ -0,0 +1,269 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Copycats rejoice! Async Clipboard API is unblocking copy & paste in Chrome 66.
+
+{# wf_updated_on: 2018-03-13 #}
+{# wf_published_on: 2018-03-13 #}
+{# wf_tags: cutandcopy,execcommand,input,chrome66 #}
+{# wf_featured_image: /web/updates/images/generic/share.png #}
+{# wf_featured_snippet: Async Clipboard API simplifies permissions-friendly copy & paste. #}
+{# wf_blink_components: Blink>Input #}
```
Comment (ebidel) [inline src/content/en/updates/2018/03/clipboardapi.md]: Let's use `Blink>DataTransfer`, same as the chromestatus entry.

File: src/data/_contributors.yaml [hunk 4]
```diff
@@ -1087,6 +1087,25 @@ josephmedley:
...
+  role:
+    - author
+    - engineer
+  homepage: 'https://jasonformat.com'
+  google: +developit
+  twitter: _developit
+  github: developit
+  email: jasonjmiller@google.com
+  description:
+    en: Jason is a Chrome DevRel
```
Comment (ebidel) [inline src/data/_contributors.yaml]: FWIW, we use "Web DevRel" now. But you can use whatever desc you want for yourself :)

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 5]
```diff
@@ -0,0 +1,269 @@
...
+<img src="/web/updates/images/2018/03/clipboardapi/prompt.png" alt="Screenshot
+of the permissions prompt shown when attempting to read from the clipboard.">
+
+
+As with anything using the Permissions API, it's possible to check if your app
+has permission to interact with the clipboard:
+
+    navigator.permissions.query({
+      name: 'clipboard-read',
+      allowWithoutGesture: false
```
Comment (ebidel) [inline src/content/en/updates/2018/03/clipboardapi.md]: keep `allowWithoutGesture: false` here?

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 6]
```diff
@@ -0,0 +1,269 @@
...
+    }).then( permissionStatus => {
```
Comment (ebidel) [inline src/content/en/updates/2018/03/clipboardapi.md]: remove space before `permissionStatus`

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 7]
```diff
@@ -0,0 +1,269 @@
...
+Prior to the introduction of the Async Clipboard API, we had a mix of different
+copy & paste implementations across web browsers.
+
+In most browsers, the browser's own copy and paste can be triggered using
+`document.execCommand('copy')` and `document.execCommand('paste')`.  If the text
+to be copied is a string not present in the DOM, we have to inject and select
+it:
+
+    button.addEventListener('click', e => {
+      let input = document.createElement('input');
```
Comment (ebidel) [inline src/content/en/updates/2018/03/clipboardapi.md]: let -> const for these

Comment (ebidel) [reply src/content/en/updates/2018/03/clipboardapi.md]: I think some of the update from the gdoc are missing? Can you double check this is the latest? 

I'll hold off on reviewing....

File: src/content/en/updates/2018/03/clipboardapi.md (hunk 2 above)
Comment (developit) [reply src/content/en/updates/2018/03/clipboardapi.md]: I copied the video embed from Surma's CSS Paint update, not sure why he did it.

File: src/content/en/updates/2018/03/clipboardapi.md (hunk 3 above)
Comment (developit) [reply src/content/en/updates/2018/03/clipboardapi.md]: Perfect, will do.

File: src/content/en/updates/2018/03/clipboardapi.md (hunk 5 above)
Comment (developit) [reply src/content/en/updates/2018/03/clipboardapi.md]: Ack! Thought I got rid of that.

- [ ] Remove reference to `allowWithoutGesture`

File: src/content/en/updates/2018/03/clipboardapi.md (hunk 7 above)
Comment (developit) [reply src/content/en/updates/2018/03/clipboardapi.md]: I had exported to markdown prior to your review and was manually syncing. I'll go over the changes and make sure no others slipped through.

Comment (developit) [reply src/content/en/updates/2018/03/clipboardapi.md]: K, I am fairly confident I've got all the changes sync'd up.

**Comment by developit** [comment]
@ebidel - sorry about that. I've gone through and made sure all the changes sync'd up.

**Comment by petele** [comment]
@ebidel - once you're happy, please 'Squash and merge'

File: src/content/en/updates/2018/03/clipboardapi.md [hunk 8]
```diff
@@ -0,0 +1,270 @@
...
+them ever knowing.
+
+As with many new APIs, [`navigator.clipboard`] is only supported for pages
+served over HTTPS. To help prevent abuse, clipboard access is only allowed when
+a page is the active tab. Pages in active tabs can write to the clipboard
+without requesting permission, but reading from the clipboard always requires
+permission.
+
+To make things easier, two new permissions for copy & paste have been added to
+the [Permissions API]. The Permissions API is available in Chrome under an
```
Comment (ebidel) [inline src/content/en/updates/2018/03/clipboardapi.md]: Permissions is on by default since 43: https://www.chromestatus.com/feature/6376494003650560

Comment (developit) [reply src/content/en/updates/2018/03/clipboardapi.md]: That was my thought as well, but the API throws when the flag is turned off. Maybe it's a bug?

**Pull request merged**